front/lib/api/assistant/configuration.ts

```diff
@@ -283,6 +283,18 @@ export async function getAgentConfigurations<V extends "light" | "full">({
               [Op.in]: Object.values(retrievalConfigs).map((r) => r.id),
             },
           },
+          include: [
```

Comment:
LGTM but aren't we supposed to run that only below line `if (variant === full)` ?

Reply:
If you expand the diff to see the beginning of the statement, you'll see that we only do this if `retrievalConfigs` is non-empty (which only happens if we're in the full variant case)

Reply:
Yep kinda hard to follow as commented on the other PR as well. But trusting you.

Reply:
IMHO when you have the full code (and not this small diff) in front of you it feels pretty obvious. I can add a comment if you feel it's useful

Reply:
I was looking at it with the full code. Thing that didn't seem obvious is why that code is not below the "if full".

Reply:
That would indent everything a little bit more to the right for now reason, given that we still need to check if there are any retrievalConfigs before fetching anyway

Reply:
Will add comments then 👍